docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,341 @@
++++
+title = "service account HTTP API "
+description = "Grafana service account HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "serviceaccount"]
+aliases = ["/docs/grafana/latest/http_api/serviceaccount/"]
++++
+
+# Service account API
+
+> If you are running Grafana Enterprise and have [Fine-grained access control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, for some endpoints you would need to have relevant permissions.
+> Refer to specific resources to understand what permissions are required.
+
+## Search service accounts with Paging
+
+`GET /api/serviceaccounts/search?perpage=10&page=1&query=myserviceaccount`
+
+#### Required permissions
```

Comment:
I'm confused about this heading. Should it be an h3 (###)? Is it really a child of the Search service accounts with Paging heading? Just curious.